### ISSUE TITLE
Switch Bypass on and off again lets Ardour crash

### ISSUE THREAD

**Issue by BeFrankly** [geraldmwangi/GuitarMidi-LV2#2]
Hello,
e used Guitar-MID in Ardour5 as a Plugin and it won't work as expected.So I played a bit with strict and not strict I/O in Ardour and while the track was in no strict I/O mode on the Plug-In GUI I enabled Bypass (so that the sound goes to the master) and when I switched of Bypass by clicking again, Ardour crashes. I guess it's got something to do with your Plug-in.

My main problem was, that the sound is not routed to the output of the track so that no sound arrives the master. See my next post.

Regards
Frank

**Comment by geraldmwangi** [comment]
It sure dies. I have been testing only with Carla. I'll check asap

**Comment by BeFrankly** [comment]
Here are some good news, I found another bug on this. As you told, you have tested with carla and yes, I also can say that switching Bypass in Carla on and off again does not let it crash. In my case I used the context menu of the plugin in the carla rack view. But! If you switch the plugin as a whole off in the rack view and switch it on again, carla crashes.
I am on Ubuntu Studio 20.04 and Carla Version 2.4.3
Regards
Frank
